lib/matplotlib/widgets.py

```diff
@@ -1332,18 +1332,13 @@ class MultiCursor(Widget):
     Example usage::
 
         from matplotlib.widgets import MultiCursor
-        from pylab import figure, show, np
```

Comment:
The bare `show` below needs to be updated.

Reply:
Handled in #11481